packages/rum/test/specs/apm-base.spec.js

```diff
@@ -330,15 +331,18 @@ describe('ApmBase', function() {
     const loggingService = serviceFactory.getService('LoggingService')
     spyOn(loggingService, 'warn')
     apmServer._makeHttpRequest = createPayloadCallback('test')
-    await apmBase.fetchCentralConfig()
-    expect(loggingService.warn).toHaveBeenCalledWith(
-      'Invalid value "NaN" for transactionSampleRate. Allowed: Number between 0 and 1.'
-    )
-    expect(configService.get('transactionSampleRate')).toBe(1)
-
-    apmServer._makeHttpRequest = createPayloadCallback('0.5')
-    await apmBase.fetchCentralConfig()
-    expect(configService.get('transactionSampleRate')).toBe(0.5)
+    apmBase.fetchCentralConfig().then(() => {
+      expect(loggingService.warn).toHaveBeenCalledWith(
+        'Invalid value "NaN" for transactionSampleRate. Allowed: Number between 0 and 1.'
+      )
+      expect(configService.get('transactionSampleRate')).toBe(1)
+
+      apmServer._makeHttpRequest = createPayloadCallback('0.5')
+      apmBase.fetchCentralConfig().then(() => {
+        expect(configService.get('transactionSampleRate')).toBe(0.5)
+        done()
```

Comment:
this test needs to account for failures. 
`done` should be handled for both success and failure cases.

Reply:
Do you mean if the promise fails? I'm returning a Promise.resolve in the lines above, so not sure how the promise could fail!